Debian GNU/kFreeBSD needs Linux-like resolver libraries to be linked into
the program.  Thanks to Axel Beckert <abe@deuxchevaux.org>.

Index: aiccu-20070115/unix-console/Makefile
===================================================================
--- aiccu-20070115.orig/unix-console/Makefile	2009-10-24 17:26:43.664837919 +0200
+++ aiccu-20070115/unix-console/Makefile	2009-10-24 17:27:05.749331932 +0200
@@ -68,6 +68,12 @@
 SRCS	+= ../common/aiccu_kame.c
 OBJS	+= ../common/aiccu_kame.o
 endif
+
+# GNU/kFreeBSD
+ifeq ($(shell uname | grep -c "kFreeBSD"),1)
+LDFLAGS += -lpthread -lresolv
+endif
+
 endif
 
 # DragonFlyBSD
